src/UI/class-settings-page.php

```diff
@@ -442,41 +453,19 @@ function (): void {
 		);
 
 		// Allow use of custom taxonomy to populate articleSection in parselyPage; defaults to category.
-		$h          = __( 'By default, Parse.ly only tracks the default post type as a post page. If you want to track custom post types, select them here!', 'wp-parsely' );
```

Comment:
Could we keep this help text (or something that's similar enough but fits with the current UI)?